backend/climateconnect_main/utility/general.py

```diff
@@ -56,3 +56,12 @@ def get_allowed_hosts(ALLOWED_HOSTS_ENV):
             if not host in allowed_hosts:
                 allowed_hosts.append(host)
     return allowed_hosts
+
+
+def get_prop_in_language(obj, prop_name: str, language_code: str) -> str:
```

Comment:
Nit: Identify type for `obj` parameter

Reply:
I couldn't find a good way to type this. These are always instances of a django model. Would you recommend to just use `obj: obj` or is there a way to do something like `obj: ModelInstance`?